crates/ruff/resources/test/fixtures/pylint/continue_in_finally.py

```diff
@@ -0,0 +1,57 @@
+while True:
+    try:
+        pass
+    finally:
+        continue  # [continue-in-finally]
+
+while True:
+    try:
+        pass
+    except Exception:
+        continue
+    finally:
+        try:
+            pass
+        finally:
+            continue  # [continue-in-finally]
+        pass
+
+
+while True:
+    try:
+        pass
+    finally:
+        continue  # [continue-in-finally]
```

Comment:
Can you add a test for nested statements?

```suggestion
    finally:
    	if True:
	        continue  # [continue-in-finally]
```

Reply:
yup, actually realized that right after pushing, better implementation incoming :)